mozregression/main.py

```diff
@@ -31,6 +33,7 @@
 from mozregression.launchers import REGISTRY as APP_REGISTRY
 from mozregression.test_runner import ManualTestRunner, CommandTestRunner
 
+LOG = getLogger(__name__.split('.')[0])
```

Comment:
You don't need this - you can just use the logger created in the cli() function.